ui/src/__snapshots__/root-reducer.test.js.snap

```diff
@@ -154,7 +154,7 @@ Object {
     "saving": false,
   },
   "pod": Object {
-    "errors": Object {},
+    "errors": null,
```

Comment:
To make the error states more generic I've set the default state to `null` which will also make for more defensive code.